packager/certcache/certcache.go

```diff
@@ -46,10 +47,19 @@ var infiniteFuture = time.Unix(1<<63-62135596801, 999999999)
 // 1MB is the maximum used by github.com/xenolf/lego/acmev2 in GetOCSPForCert.
 // Alternatively, here's a random documented example of a 20K limit:
 // https://www.ibm.com/support/knowledgecenter/en/SSPREK_9.0.0/com.ibm.isam.doc/wrp_stza_ref/reference/ref_ocsp_max_size.html
-const maxOCSPResponseBytes = 1024 * 1024
+const MaxOCSPResponseBytes = 1024 * 1024
 
 // How often to check if OCSP stapling needs updating.
-const ocspCheckInterval = 1 * time.Hour
+const OcspCheckInterval = 1 * time.Hour
+
+// How often to check if certs needs updating.
+const CertCheckInterval = 24 * time.Hour
+
+// Recommended renewal duration for certs.
```

Comment:
Maybe clarify in the comments that this is the duration before the next expiry rather than  the duration after the last renewal.

Reply:
Done.